src/main/java/frc/robot/commands/assist/ScorePiece.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands.assist;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class ScorePiece extends SequentialCommandGroup {
+  // TODO: read intake voltage and read piece sensor
```

Comment:
Expected behavior:

Assumptions:
- Arm Angle and Shooter Speed are ready for scoring

Score Piece:
1. Turn on intake (to feed piece into shooter)
2. Wait for piece to be shot out
3. Turn **off** intake
4. Turn **off** shooter